.github/workflows/release.yml

```diff
@@ -142,29 +142,36 @@ jobs:
           cp kframework_amd64_debian_bookworm.deb kframework_${version}_amd64_debian_bookworm.deb
           gh release upload --repo runtimeverification/k --clobber v${version} kframework_${version}_amd64_debian_bookworm.deb
 
-  arch:
-    name: 'Arch Linux Package'
-    runs-on: [self-hosted, linux, normal]
-    timeout-minutes: 90
-    steps:
-      - uses: actions/checkout@v3
-      - name: 'Build and Test Package'
-        uses: ./.github/actions/test-package
-        with:
-          dockerfile: package/arch/Dockerfile
-          os: archlinux
-          distro: base
-          build-package: package/arch/build-package
-          test-package: package/arch/test-package
-          pkg-name: kframework_arch_x86_64.pkg.tar.zst
-      - name: 'Upload Package to Release'
-        env:
-          GITHUB_TOKEN: ${{ secrets.JENKINS_GITHUB_PAT }}
-        run: |
-          set -x
-          version=$(cat package/version)
-          cp kframework_arch_x86_64.pkg.tar.zst kframework_${version}_arch_x86_64.pkg.tar.zst
-          gh release upload --repo runtimeverification/k --clobber v${version} kframework_${version}_arch_x86_64.pkg.tar.zst
+  # The arch linux packaging step is disabled because of recurrent flakiness on
+  # the runners we use:
+  #   - https://github.com/runtimeverification/k/issues/3723
+  #   - https://github.com/runtimeverification/k/issues/3731
```

Comment:
You need to also change the `release` step to not have `needs: [arch]`.

Reply:
Good catch, thanks!